Formula/libedit.rb

```diff
@@ -26,6 +27,13 @@ def install
                           "--disable-silent-rules",
                           "--prefix=#{prefix}"
     system "make", "install"
+
+    on_linux do
+      # Conflicts with ncurses.
+      mv man3/"history.3", man3/"history_libedit.3"
+      # Symlink libedit.so.0 to libedit.so.2 for binary compatibility with Debian/Ubuntu.
+      ln_s lib/"libedit.so.0", lib/"libedit.so.2"
```

Comment:
Can we check if this is still needed, and what issue it solves? Maybe some git blame and research will be needed.

Reply:
It was added here: https://github.com/Homebrew/linuxbrew-core/commit/6a5dec806517998736a4da89f655490657d45b84#diff-b2ce0a987a063141bb35c3c277caf09d0d90ac5385418b5d79a9275935dbe816.  @sjackman can you explain what this was for and how we can check if it's still needed?

Reply:
https://packages.debian.org/buster/amd64/libedit2/filelist

Debian/Ubuntu patched the source to change the `SONAME` from `0` to `2`.
https://packages.debian.org/buster/amd64/libedit2/filelist

This patch was likely so that we could use some binary bootstrap executable that was dynamically linked against `libedit.so.2`. I'm sorry that I didn't record which in the commit log message.

Here's the upstream patch:
https://sources.debian.org/patches/libedit/3.1-20181209-1/update-soname.diff/

I wanted to find out what other distributions do. Repology seems to track source package names rather than binary package names so wasn't entirely helpful: https://repology.org/project/libedit/versions

It looks like Fedora uses a `SONAME` of 0. https://src.fedoraproject.org/rpms/libedit/blob/rawhide/f/libedit.spec

I'm inclined to drop this patch until we find a need for it

Reply:
I see that Arch Linux also uses only `libedit.so.0`.  I agree with @sjackman that this fix should be removed for now until we have a clear reason to keep it.  I will remove that in linuxbrew-core and see if anything breaks, and only commit the file conflict fix here.